src/beeware_theme/_static/beeware-theme.css

```diff
@@ -34,6 +46,19 @@ body[data-theme="light"] {
     font-family: 'Cutive', serif;
 }
 
+.sidebar-drawer {
+    font-size: 110%;
+}
+
+.main {
+    box-sizing: unset;
```

Comment:
It's possible to guess what most of the other rules are for, but this one should have a comment explaining it.